### PR TITLE
SAR-10327 Update messages file

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -97,7 +97,7 @@ error.no_entry_sa-postcode                                   = Enter the postcod
 
 ## Overseas Tax Identifier Page
 tax-identifier.title                                         = Does the business have an overseas tax identifier?
-tax-identifier.line_1                                        = We may use this number to help us identify your business. The tax identifier could be a VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify your business details.
+tax-identifier.hint                                          = This may help us identify the business. It could be a Vat registration number, Employee Idenfication Number (EIN) or any other identifier we could use to verify the business details.
```
Comment (slavvco) [inline conf/messages:100]: please check the original ticket, there is a small change Vat -> VAT that needs to be done, in test below as well

**Review by slavvco — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (refaque-ahmad) [reply conf/messages:100]: Just a small comment regarding the typo mistake I found in the ticket when doing ME is >>> "Idenfication"

Comment (slavvco) [reply conf/messages:100]: thanks @refaque-ahmad

**Review by slavvco — APPROVED** [review]
Looks good.

**Pull request merged**